server/src/api/controllers/commandController.js

```diff
@@ -48,7 +48,9 @@ export async function newCommand(req, res) {
 
   await c.save()
 
-  const response = await commandHandler(session, c, additionalData)
+  const commandHandler = new CommandHandler(session, c, additionalData)
+
+  const response = await commandHandler.run()
```

Comment:
Why not await here? What's the difference between await and new?

Reply:
You can see the constructor definition in commandHandler.js. Constructor is the initialization function of a instance of class, which cannot be asynchronize, so I split the async code to a async method.
You need to re-cap the object oriented design.